sdk/python/feast/infra/online_stores/contrib/contrib_repo_configuration.py

```diff
@@ -0,0 +1,10 @@
+from tests.integration.feature_repos.integration_test_repo_config import (
+    IntegrationTestRepoConfig,
+)
+from tests.integration.feature_repos.universal.online_store.hbase import (
+    HbaseOnlineStoreCreator,
+)
+
+FULL_REPO_CONFIGS = [
```

Comment:
mind rebasing against master? there's now a postgres folder in here too. maybe make this hbase_repo_configuration?

Reply:
Done!